wandb/sdk/wandb_init.py

```diff
@@ -180,13 +173,17 @@ def setup(self, kwargs) -> None:  # noqa: C901
         config_include_keys = kwargs.pop("config_include_keys", None)
         config_exclude_keys = kwargs.pop("config_exclude_keys", None)
 
-        # todo: deprecate config_include_keys and config_exclude_keys
-        # if config_include_keys or config_exclude_keys:
-        #     wandb.termwarn(
-        #       "config_include_keys and config_exclude_keys are deprecated:"
-        #       " use config=wandb.helper.parse_config(config_object, include=('key',))"
-        #       " or config=wandb.helper.parse_config(config_object, exclude=('key',))"
-        #     )
+        deprecated_kwargs = {
+            "config_include_keys": (
+                "Use `config=wandb.helper.parse_config(config_object, include=('key',))` instead."
+            ),
+            "config_exclude_keys": (
+                "Use `config=wandb.helper.parse_config(config_object, exclude=('key',))` instead."
+            ),
+        }
+        for deprecated_kwarg, msg in deprecated_kwargs.items():
+            if locals().get(deprecated_kwarg):
```

Comment:
something like this?
```suggestion
        local_args = dict(locals())
        for deprecated_kwarg, msg in deprecated_kwargs.items():
            if deprecated_kwarg in local_args:
```